src/main/java/internetShop/Product.java

```diff
@@ -0,0 +1,47 @@
+package internetShop;
+
+//@Data
+public class Product {
+    private String name;
+    private long price;
+    private Category category;
+
+    public Product(String name, long price, Category category) {
+        this.name = name;
+        this.price = price;
+        this.category = category;
+    }
+
+    public String getName() {
+        return name;
```

Comment:
reduce all the get/set toString Equals
by Lombok
@Getter/@Setter/@ToString/@EqualsAndHashcode/@NoArgsConstructor/@RequiredArgsConstructor and etc